pegasus/sites.v3/code.org/public/css/maker-skinny-banner.scss

```diff
@@ -0,0 +1,152 @@
+@import "color.scss", "font.scss", "imports/_breakpoints.scss";
+
+.maker-skinny-banner {
+  box-sizing: content-box;
+  height: 130px;
+  overflow: hidden;
+  padding: 0 0 0 1em;
+  background: #00736E;
+  position: relative;
+
+  @media screen and (max-width: $width-md) {
+    padding: 0 1.5em;
+  }
+
+  @media screen and (max-width: $width-sm) {
+    height: 230px;
+    padding: 1.5em 1.15em 0.5em;
+    margin: 3em auto 0;
+  }
+
+  @media screen and (max-width: $width-xs) {
+    height: 260px;
+  }
+
+  a:hover,
+  a:active,
+  a:visited {
+    text-decoration: none;
+  }
+
+  .wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    max-width: 1020px;
+    margin: 0 auto;
+
+    @media screen and (max-width: $width-md) {
+      width: 72%;
+      margin: 0;
+    }
+
+    @media screen and (max-width: $width-sm) {
+      justify-content: center;
+      width: 100%;
+      margin-top: -1em;
+    }
+  }
+
+  img.left {
+    align-self: start;
+
+    @media screen and (max-width: $width-md) {
+      display: none;
+    }
+
+    @media screen and (max-width: $width-sm) {
+      display: block;
+      position: relative;
+      top: -8px;
+    }
+  }
+
+  img.right {
+    align-self: end;
+
+    @media screen and (max-width: $width-md) {
+      position: absolute;
+      z-index: 0;
+      right: 0;
+    }
+
+    @media screen and (max-width: $width-sm) {
+      display: none;
+    }
+  }
+
+  // There's a mysterious <p> tag being added around
+  // the right image on https://code.org/circuitplayground.
+  // This aligns it correctly to the bottom of the container.
+  p img {
```

Comment:
If the <p> tag is not needed, could it be removed so that we don't need absolute position correcting?

Reply:
I don't know where that `<p>` tag is coming from 😅 It's not in the original code, and doesn't show up on the /applab page. I assume there's some automated process that's adding it when the site builds, but that's beyond my understanding.